src/DDTrace/Integrations/Laravel/LaravelIntegration.php

```diff
@@ -88,18 +85,15 @@ function ($This, $scope, $args, $route) use ($rootSpan, $integration) {
                 list($request) = $args;
 
                 // Overwriting the default web integration
-                $integration->addTraceAnalyticsIfEnabledLegacy($rootSpan);
+                $integration->addTraceAnalyticsIfEnabled($rootSpan);
                 $routeName = LaravelIntegration::normalizeRouteName($route->getName());
 
-                $rootSpan->setTag(
-                    Tag::RESOURCE_NAME,
-                    $route->getActionName() . ' ' . $routeName
-                );
+                $rootSpan->resource = $route->getActionName() . ' ' . $routeName;
 
-                $rootSpan->setTag('laravel.route.name', $routeName);
-                $rootSpan->setTag('laravel.route.action', $route->getActionName());
-                $rootSpan->setTag('http.url', $request->url());
-                $rootSpan->setTag('http.method', $request->method());
+                $rootSpan->meta['laravel.route.name'] = $routeName;
+                $rootSpan->meta['laravel.route.action'] = $route->getActionName();
+                $rootSpan->meta['http.url'] = $request->url();
```

Comment:
Don't miss this one, for _reasons_ this does not use Tag::HTTP_URL, but the string value...

Reply:
Fixed https://github.com/DataDog/dd-trace-php/pull/1383/commits/7e375b84c45fa00e15853a4fcbb17e3ef1dc6b06